src/components/Carousel.vue

```diff
@@ -188,9 +188,12 @@ export default {
     padding: 0;
     margin: 0;
     background: black;
+    width: 95%;
+    max-width: 900px;
```

Comment:
Woke up and saw this. To be honest I don't like the approach of setting a fixed max-width for the entire room. I'm trying to work on a solution for this right now based on your solution which in overall is pretty intresting